error: non-pointer operand type 'unsigned int' incompatible with nullptr

```
/wrkdirs/usr/ports/biology/py-python-libsbml/work-py39/python-libsbml-5.20.0/libsbml_source/src/sbml/SBase.cpp:1090:29: error: non-pointer operand type 'unsigned int' incompatible with nullptr
  return (mHistory != NULL) ? mHistory->getNumModifiedDates() : NULL;
                            ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

NULL is not properly defined in C++. In fact, on some systems it is defined as a zero-char. ```nullptr``` should be used instead as a null pointer.

Version: 5.20.0
clang-14
FreeBSD 13.1

--- a/src/sbml/SBase.cpp
+++ b/src/sbml/SBase.cpp
@@ -1087,7 +1087,7 @@ SBase::getModifiedDate(unsigned int n)
 unsigned int
 SBase::getNumModifiedDates()
 {
-  return (mHistory != NULL) ? mHistory->getNumModifiedDates() : NULL;
+  return (mHistory != NULL) ? mHistory->getNumModifiedDates() : 0;
 }
 
 

